internal/secret/kubernetes.go

```diff
@@ -70,7 +71,9 @@ func CreateKubeSecret(req KubeSecretRequest) (v1.Secret, error) {
 		}
 	} else {
 		for key, specItem := range req.Spec {
-			if specItem.Source != "" { // Map one secret
+			if specItem.Value != "" { // Map value directly
```

Comment:
I think the value should be the last checked item. Partly to maintain backward compatibility, partly because an empty value is/can be a valid value, but empty secret references cannot.